solidity/dashboard/src/components/WithdrawETHModal.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from "react"
+import { getErrorsObj } from "../forms/common-validators"
+import { withFormik } from "formik"
+import web3Utils from "web3-utils"
+import { useWeb3Context } from "./WithWeb3Context"
+import { tbtcAuthorizationService } from "../services/tbtc-authorization.service"
+import { useShowMessage, messageType } from "./Message"
+import * as Icons from "./Icons"
+import AvailableEthAmount from "./AvailableEthAmount"
+import AvailableETHForm from "./AvailableETHForm"
+
+const WithdrawETHModal = ({ operatorAddress, availableETH, closeModal }) => {
+  const web3Context = useWeb3Context()
+  const { web3 } = web3Context
+  const showMessage = useShowMessage()
+
+  const onSubmit = useCallback(
+    async (formValues, onTransactionHashCallback) => {
+      const { ethToWithdraw: ethToWithdraw } = formValues
```

Comment:
Unnecessary. Just `const { ethToWithdraw } = formValues`

Reply:
yea, refactored and it's gone now